### ISSUE TITLE
Invalid suggestions are given for some misspellings

### ISSUE THREAD

**Issue by rotemdan** [swyphcosmo/vscode-spellchecker#28]
This happened several times before but I didn't keep the particular misspellings that triggered it. This time I did.

Create a new text document. Paste:
```
correpsonding
```
The suggestion given by the spellchecker is:
```
[Spell Checker] Spelling [ correpsonding ]: suggestions [ correspondin ]
```
The light bulb shows `correspondin` as a suggested correction. Which is not a correct spelling.

Correcting with `correspondin` the spellchecker shows the suggestion:
```
[Spell Checker] Spelling [ correspondin ]: 
suggestions [ corresponds, corresponding, correspond, corresponde ]
```
Notice one of the suggestions "`corresponde`" is also an incorrect spelling.

I'm not sure if this is a bug in the extension or the underlying spelling library but I thought this was common enough it should be reported here.

(Using 1.2.8 on latest VSCode Insider Win10 x64,  empty `settings.json` and no spellchecker.json in the workspace)

**Comment by rotemdan** [comment]
Another example:
```
percisely
```
Correction message:
```
[Spell Checker] Spelling [ percisely ]: suggestions [ precisel ]
```
Seems like the last character of the correction word is omitted for some reason?

**Comment by swyphcosmo** [comment]
That's exactly what's happening. It's interesting that I never caught this before.  Thanks! I'll push an update soon.
